src/styles/_catalog.scss

```
@@ -1,22 +1,32 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+$cardWidth: 200px;
+$cardHeight: 300px;
 
-  max-width: 944px;
-  margin: 0 auto;
+.catalog {
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  grid-template-columns: repeat(1, $cardWidth);
```

no need to repeat 1 column
```suggestion
  grid-template-columns: $cardWidth;
```